addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PersistentXSSStorage.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2013 The ZAP Development Team
```

Comment:
2020?

Reply:
I mentioned this a number of times, the files appear completely new instead of just renamed for some reason. If the content hasn't changed much then the original date(s) should be retained I guess.

Reply:
Thx, I will take that into consideration in the future